properties/P000061.md

```diff
@@ -2,8 +2,27 @@
 uid: P000061
 name: Cozero complemented
 refs:
-  - mr: 2247908
-    name: Products of cozero complemented spaces (G. Gruenhage)
+  - zb: "1067.54015"
+    name: Cozero complemented spaces; when the space of minimal prime ideals of a $C(X)$ is compact (Henriksen & Woods)
 ---
 
-A {P6} space in which for every cozero set $U$ there is a cozero set $V$ disjoint from $U$ such that $U \cup V$ is dense in $X$.
+Every cozero set $U$ in $X$ has a (not necessarily unique) *cozero complement*, that is,
```

Comment:
```suggestion
Every cozero set $U$ in $X$ has a *cozero complement*, that is,
```

I wouldn't think anyone would jump to uniqueness here, but it's fine to keep as well.